database/migrations/2024_06_25_101545_change_slug_in_case_studies_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Are these remnants from another branch?

Reply:
fixed, came from previous pull request (before your changes in code)